### ISSUE TITLE
quat.fromEuler uses degrees

### ISSUE THREAD

**Issue by magcius** [toji/gl-matrix#410]
As far as I can tell, this is the only API that seems to use degrees, which is bizarre. Everywhere else uses radians. It's documented, but it was a bit of a footgun for me.

**Comment by stefnotch** [comment]
That is true, depending on how much attention this issue gets, we might want to change that.

For the record, there are other APIs that use degrees for euler angles and radians for pretty much everything else, like [Unity's API](https://docs.unity3d.com/ScriptReference/Quaternion.Euler.html)

There is one other API that uses degrees, because it's [explicitly designed for WebVR](https://github.com/toji/gl-matrix/blob/5e3d5a6e29fcecbb2c861266691c3f41b7b81001/src/mat4.js#L1577-L1580)

**Comment by toji** [comment]
Worth noting that the WebVR is deprecated now and WebXR (it's replacement) no longer reports FOVs directly, so `perspectiveFromFieldOfView`'s use cases are likely quite limited now.
